### PR TITLE
Vmemcache doesn't support fiber cache compression

### PR DESCRIPTION
**Pull request by jikunshang**
## What changes were proposed in this pull request?
Vmemcache doesn't support fiber cache compression, will throw an Exception([Issue 1186](https://github.com/Intel-bigdata/OAP/issues/1186) )

### CODE REVIEW

File: src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala [hunk 1]
```diff
@@ -487,6 +487,10 @@ class VMemCache(fiberType: FiberType) extends OapCache with Logging {
       val length = res
       val fiberCache = emptyDataFiber(length)
       val startTime = System.currentTimeMillis()
+      if( OapRuntime.getOrCreate.fiberCacheManager.dataCacheCompressEnable ) {
```
Comment (winningsix) [inline src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala:490]: Could we just change L454 by introducing ```require(OapRuntime.getOrCreate.fiberCacheManager.dataCacheCompressEnable, "Vmemcache strategy doesn't support fiber cache compression" +
          " currently, please try other strategy.")```?

Comment (jikunshang) [reply src/main/scala/org/apache/spark/sql/execution/datasources/oap/filecache/OapCache.scala]: Add this when initialize vmemcache

**Comment by winningsix** [comment]
LGTM. 

@zhixingheyi-tian @HongW2019  is there any place that FiberCache can bring some performance benefits? 

@jikunshang what's the performance degradation we observed in our test?

**Comment by winningsix** [comment]
cc @offthewall123

**Comment by winningsix** [comment]
This closes #1186

**Pull request merged**